normandy/bundler/tests/test_serializers.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from normandy.bundler.tests import BundleFactory
+from normandy.bundler.serializers import BundleSerializer
+from normandy.recipes.tests import RecipeFactory
+
+
+@pytest.mark.django_db()
+def test_bundle_serializer():
+    recipe = RecipeFactory()
+    bundle = BundleFactory(recipes=[recipe])
+    serializer = BundleSerializer(bundle)
+
+    assert len(serializer.data['recipes']) == 1
+    assert serializer.data['recipes'][0] == {'name': recipe.name, 'actions': []}
```

Comment:
I see now that it would be smaller that way. I used to have `assert serializer.data == ...`, which was really verbose, and this was simpler. Asserting the list directly would be better now.